src/OrchardCore/OrchardCore.Data.YesSql/Removing/ShellDbTablesInfo.cs

```diff
@@ -66,17 +67,30 @@ public IEnumerable<string> GetTableNames()
     }
 
     public ISchemaBuilder CreateMapIndexTable(Type indexType, Action<ICreateTableCommand> table, string collection)
```

Comment:
How about adding `[Obsolete]` on these sync-over-async methods?

Reply:
The interface already has the attribute